src/main/java/pl/javastart/task/SampleClass.java

```diff
@@ -0,0 +1,23 @@
+package pl.javastart.task;
+
+public class SampleClass {
+
+    final static double PI = 3.14;
```

Comment:
checkstyle. Raczej zapisujemy `static final`, a nie `final static`. Nie ma to wpływu na działanie